Framework/Algorithms/src/SofQWNormalisedPolygon.cpp

```diff
@@ -165,20 +379,22 @@ void SofQWNormalisedPolygon::exec() {
       const double dE_jp1 = X[j + 1];
 
       const double lrQ = m_EmodeProperties.q(dE_jp1, thetaLower, det);
+
       const V2D ll(dE_j, m_EmodeProperties.q(dE_j, thetaLower, det));
       const V2D lr(dE_jp1, lrQ);
       const V2D ur(dE_jp1, m_EmodeProperties.q(dE_jp1, thetaUpper, det));
       const V2D ul(dE_j, m_EmodeProperties.q(dE_j, thetaUpper, det));
       if (g_log.is(Logger::Priority::PRIO_DEBUG)) {
-        logStream << "Spectrum=" << specNo << ", theta=" << theta
-                  << ",thetaWidth=" << thetaWidth << ". QE polygon: ll=" << ll
-                  << ", lr=" << lr << ", ur=" << ur << ", ul=" << ul << "\n";
+        logStream << "Spectrum=" << specNo
+                  << ", lower theta=" << thetaLower * rad2deg
+                  << ", upper theta=" << thetaUpper * rad2deg
+                  << ". QE polygon: ll=" << ll << ", lr=" << lr << ", ur=" << ur
+                  << ", ul=" << ul << "\n";
       }
 
       using FractionalRebinning::rebinToFractionalOutput;
-      rebinToFractionalOutput(Quadrilateral(std::move(ll), std::move(lr),
```

Comment:
I assume the moves of the V2D made very little difference to the performance?

Reply:
I might be mistaken but I don't think any moves took place here.
1. `Quadrilateral`'s constructror takes its arguments by normal reference
2. `ll`, `lr`, `ur` and `ul` are marked as `const` above
3. What is there to move in `V2D`?

Reply:
Ah I assumed `Quadrilateral` took the arguments by value. Never mind :)